src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const PRICE_IN_DAY = 40;
+
+  let price = days * PRICE_IN_DAY;
+
+  if (days >= LONG_TERM) {
+    price = price - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    price = price - SHORT_TERM_DISCOUNT;
```

Comment:
The second `if` condition is unnecessary because if `days` is greater than or equal to `LONG_TERM`, the function would have already returned. You can remove the `days < LONG_TERM` check.